### PR TITLE
Add GESTE and GESTE_Examples

### PR DESCRIPTION
**Pull request by Fabien-Chouteau**

### CODE REVIEW

File: index/ge/geste_examples.toml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+                 "RPG/RPG.gpr",
+                 "racing/racing.gpr"]
+
+licenses = ["BSD 3-Clauses"]
+
+['1.0.0']
+origin = 'https://github.com/Fabien-Chouteau/GESTE-examples/releases/download/v1.0.0/GESTE_Examples-1.0.0.zip'
+origin-hashes = ['sha512:2d94ac6de0a7d5c7087d46d3f276ed69e35b3b22b9322a7e53031ec77a18e46b90162372182401ee4078923ffefb884eb2f1c11a6c00c7727a65e7ee4fe6fe83']
+['1.0.0'.depends-on]
+geste = "1.0.0"
```
Comment (mosteo) [inline index/ge/geste_examples.toml]: I guess the use of version-exact dependencies here is intended?

**Comment by mosteo** [comment]
The examples look gorgeous, BTW.

File: index/ge/geste_examples.toml (hunk 1 above)
Comment (Fabien-Chouteau) [reply index/ge/geste_examples.toml]: I didn't think this through, in theory the examples should work on any GESTE 1.x.x.

**Comment by Fabien-Chouteau** [comment]
> The examples look gorgeous, BTW.

Thanks to the authors of the sprites listed in the READMEs :)

File: index/ge/geste_examples.toml (hunk 1 above)
Comment (mosteo) [reply index/ge/geste_examples.toml]: Actually this is default that makes me a bit uncomfortable: a lack of operator is interpreted as "=". Rust, for example, defaults to "^" ([here](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html)). If we required an explicit operator, however, equility results in ugly `geste = "=1.0.0"` (this is already accepted btw).

**Pull request merged**